third_party/jaro_winkler/details/intrinsics.hpp

```diff
@@ -95,10 +95,10 @@ static inline int tzcnt(uint64_t x)
 #    endif
 
 #else /*  gcc / clang */
-static inline int tzcnt(uint32_t x)
-{
-    return __builtin_ctz(x);
-}
+//static inline int tzcnt(uint32_t x)
```

Comment:
Should this be commented out? 
If yes, why not remove it?

Reply:
Yes, this breaks the MacOS CI build. I would rather not remove it since it is in a third party library.